File: PlanlogicAI/ClientApp/src/app/models/Liability.ts
export interface Liability {

    liabilityId: number,
    clientId: number,
    type:string,
    name: string,
    deductibility : number,
    owner: string,
    principal: number,
    repaymentType : string,
    repayment : number,
    interestRate: number,
    term: number,
    commenceOnDateType: string,
    commenceOnDate: number,
    repaymentDateType: string,
    repaymentDate: number,
    associatedAsset: string,
    creditLimit: number
}

export interface LiabilityDetails {

    liabilityId: number,
    clientId: number,
    amount: number,
    fromDateType: string,
    fromDate: number,
    toDateType: string,
    toDate: number
  
}

File: PlanlogicAI/Data/Liability.cs
using System;
using System.Collections.Generic;

namespace PlanlogicAI.Data
{
    public partial class Liability
    {
        public int LiabilityId { get; set; }
        public int ClientId { get; set; }
        public string Type { get; set; }
        public string Name { get; set; }
        public int Deductibility { get; set; }
        public string Owner { get; set; }
        public int Principal { get; set; }
        public string RepaymentType { get; set; }
        public int Repayment { get; set; }
        public decimal InterestRate { get; set; }
        public int Term { get; set; }
        public string CommenceOnDateType { get; set; }
        public int? CommenceOnDate { get; set; }
        public string RepaymentDateType { get; set; }
        public int? RepaymentDate { get; set; }
        public string AssociatedAsset { get; set; }
        public int CreditLimit { get; set; }
    }
}
